Tests/AsyncHTTPClientTests/HTTPClientTestUtils.swift

```diff
@@ -234,7 +236,7 @@ internal final class HTTPBin {
 
         self.serverChannel = try! ServerBootstrap(group: self.group)
             .serverChannelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
-            .childChannelOption(ChannelOptions.socket(IPPROTO_TCP, TCP_NODELAY), value: 1)
+            .childChannelOption(ChannelOptions.socket(IPPROTO_TCP, TCP_NODELAY), value: usingTCP ? 1 : 0)
```

Comment:
Rather than do this, we should probably just only set this conditionally on whether we're using TCP. The kernel is within its rights to refuse _any_ manipulation of `TCP_NODELAY` on a unix socket, even turning it off.

Reply:
We can delete this line without replacement. NIO in the required version _defaults_ to `TCP_NODELAY`.

Reply:
> We can delete this line without replacement. NIO in the required version _defaults_ to `TCP_NODELAY`.
I'll go ahead and remove this line, undo the rest of the changes here, and mention this in the commit.